imagehash/__init__.py

```diff
@@ -32,9 +32,23 @@
 from __future__ import (absolute_import, division, print_function)
```

Comment:
Is it not possible to remain with imagehash.py?

I see that PEP 561 does not support single-file packages. But would the "old" way of distributing (PEP 484) work here, since there is no actual type stub? Or a -stubs package?

Reply:
You need *either* a `py.typed` or stubs (`.pyi` file(s))for type checkers to not whine about missing stubs.
If you'd prefer using stubs and staying as a single-file module, then #161 would be the way to go (although that PR has a few things that should be updated)